tools/swift-ast-script/swift-ast-script.cpp

```diff
@@ -62,7 +62,8 @@ class Observer : public FrontendObserver {
 
 }
 
-int main(int argc, const char *argv[]) {
+// ISO C++ does not allow 'main' to be used by a program [-Wmain]
```

Comment:
Huh, never knew about this.

Reply:
Ya, clang top-of-tree is now warning about this by default.